subprojects/xcfa/xcfa/src/main/java/hu/bme/mit/theta/xcfa/passes/XcfaPassManager.java

```diff
@@ -83,7 +83,7 @@ public class XcfaPassManager {
 //				new AssignmentChainRemoval(),
 //				new NoReadVarRemovalPass(),
 //				new GlobalVarsToStoreLoad(),
-				new UnusedVarRemovalPass(),
+//				new UnusedVarRemovalPass(),
```

Comment:
This is a global modification that will also affect C source codes. My questions are:

1. Why was this modification needed?
2. If the change is needed for CHCs, can we somehow make XcfaPassManager configurable to include the pass for C codes and remove it for CHCs.

Reply:
The modification was necessary because the pass removed variables that would've been necessary for correct verification, resulting in incorrect answers. I added methods to XcfaPassManager that can remove passes and moved the removal to CHC handling part of XcfaCli.